packages/confirm/jest.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Copy https://github.com/MetaMask/metamask-module-template/blob/main/jest.config.js into root of repo. add property `"root": true` and extend it in every package